README.md

```diff
@@ -963,13 +972,15 @@ INFO     Removing star/align
 INFO     Successfully removed star/align module
 ```
 
+You can pass the module name as a optional argument to `nf-core modules install` instead of using the cli prompt, eg: `nf-core modules remove fastqc`. To specify the pipeline directory, use `--dir <pipeline_dir>`.
```

Comment:
```suggestion
You can pass the module name as an optional argument to `nf-core modules install` instead of using the cli prompt, eg: `nf-core modules remove fastqc`. To specify the pipeline directory, use `--dir <pipeline_dir>`.
```